### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by skroczek**
This pull request makes this library compatible to the new versions of the dependencies (symfony/routing > 2.2 and Twig > 2.0).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
     "require": {
         "jms/metadata": "1.*",
-        "symfony/property-access": "~2.2"
+        "symfony/property-access": ">=2.2"
```
Comment (staabm) [inline composer.json]: you should not define version constraints without a upper bound

Comment (staabm) [inline composer.json]: better explicitly state when this lib is compatible with several major versions, e.g. `~2.2||~3.0`

Comment (skroczek) [inline composer.json]: No problem I will fix this.

**Comment by schmittjoh** [comment]
Could you rebase this one, it seems to have conflicted with one of the other PRs.

**Comment by skroczek** [comment]
Thanks @schmittjoh, I rebased this pull request.

**Pull request merged**